<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>438</width>
    <height>409</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>438</width>
    <height>409</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Секундомер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="L_time">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>Время</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="TB_Krug">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_Start">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_Clear">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_Krug">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Круг</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>PB_Start</tabstop>
  <tabstop>PB_Clear</tabstop>
  <tabstop>PB_Krug</tabstop>
  <tabstop>TB_Krug</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
